acapy_agent/protocols/present_proof/v2_0/formats/anoncreds/handler.py

```diff
@@ -175,7 +176,7 @@ async def create_pres(
     async def receive_pres(self, message: V20Pres, pres_ex_record: V20PresExRecord):
         """Receive a presentation and check for presented values vs. proposal request."""
 
-        def _check_proof_vs_proposal():
```

Comment:
This class should really get some refactoring but don't want to address it in this PR.